packages/commands/src/index.ts

```diff
@@ -579,6 +594,36 @@ export class CommandRegistry {
     this._startTimer();
   }
 
+  /**
+   * Process a ``keyup`` event to clear the timer on the modifier id exists.
```

Comment:
what is `id` here?

Reply:
typo:

`Process a ``keyup`` event to clear the timer on the modifier, if it exists`